openwisp-qa-check

```diff
@@ -60,6 +68,26 @@ runcheckendline() {
   fi
 }
 
+runstylelintcheck() {
+  stylelint $(find . -type f -name "*.css") &&
```

Comment:
what happens if `stylelint` is not installed? Can you make sure the function skips with a warning?
We need this to avoid breaking existing builds which are using the master version of this module.

Reply:
Added a check for this